Make.defs

```diff
@@ -108,10 +108,6 @@ define REGISTER
 	$(Q) touch "$(BUILTIN_REGISTRY)$(DELIM).updated"
 endef
 
-define ARLOCK
-	$(Q) flock $1.lock $(call ARCHIVE, $1, $(2))
```

Comment:
Hmmmm flock was necessary to get the nightly build in a stable situation.  @xiaoxiang781216 could you please comment here.